### DISCUSSION TITLE
Activating dsub with google assistant

### DISCUSSION THREAD

**Discussion by phirestalker** [daneren2005/Subsonic#1123] [General]
I have been trying to set dsub as my default music provider. After removing ALL of my other music apps the phone FINALLY brought up choices to use for my default music app. Unfortunately, dsub is not listed. Google says I can just say the name of any app. However, when I say dsub it has d-sub which causes it not to be found.

How do I say the name of the app so Google Assistant will recognize it.

I have tried typing in my response, but that results in a whole new query to Google Assistant instead of an answer to its question.

**Comment by FacialLeather** [comment]
I tried this several years ago, I figured it just dsub want up-to-date enough or coded correctly (not that I would I would know as I know nothing about coding. 😉) for Google to understand. If Scott decides to change it then I hope I hear about it through your thread. I assume the Google default players are major streaming platforms only. Maybe those apps pay Google to be there.
